cnxepub/collation.py

```diff
@@ -33,7 +33,8 @@ def easybake(ruleset, in_html, out_html):
     with respective read and write ability.
 
     """
-    html = etree.parse(in_html)
+    html_parser = etree.HTMLParser(encoding="utf-8")
```

Comment:
Was the PR description tl;dr? It's nothing but a description of the problem and why this workaround was chosen over rewriting the selector parser in easybake.